python/cuml/test/test_tsne.py

```diff
@@ -28,7 +28,7 @@ def test_tsne(name):
     X = eval("datasets.load_{}".format(name))().data
     X_cudf = cudf.DataFrame.from_pandas(pd.DataFrame(X))
 
-    for i in range(3):
+    for i in range(100):
```

Comment:
This change adds **a lot** of running time, no? In my quick comparison against other PR like half an hour, which given that we are already are at really bad PR CI times, is definitely something we cannot have for a single test. Please change this.

Reply:
We should pass this in as a parameter and set it to 3 for regular tests and 100-200 for stress tests.

Reply:
Ohhh oopss! I forgot to change it back...

Reply:
Loll originally it was 3 but just for confirmation purposes I set it temporarily to 100

Reply:
Oh interesting idea @cjnolet! I guess I can add it as a stress param. But I guess for now it seems like 10 times in a row after running it 100 times for each one, there are no memory leaks!

So maybe for now 3 is ok? (it's more like 12 iters since each run runs TSNE 4 times) (Since before even 3 wasn't good enough).